.aws/task-definition-template.json

```diff
@@ -0,0 +1,187 @@
+        },
+        {
+          "name": "SLACK_NOTIFICATION_CHANNEL",
+          "valueFrom": "slack-notification-channel-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_CRON_PERSONOWNERREBUILDJOB",
+          "valueFrom": "person_owner_rebuild_job_cron-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_CRON_PERSONPLACEMENTEMPLOYINGBODYTRUSTJOB",
+          "valueFrom": "person_placement_employing_body_trust_job_cron-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_CRON_PERSONPLACEMENTTRAININGBODYTRUSTJOB",
+          "valueFrom": "person_placement_training_body_trust_job_cron-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_CRON_POSTEMPLOYINGBODYTRUSTJOB",
+          "valueFrom": "post_employing_body_trust_job_cron-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_CRON_POSTTRAININGBODYTRUSTJOB",
+          "valueFrom": "post_training_body_trust_job_cron-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_CRON_PERSONELASTICSEARCHJOB",
+          "valueFrom": "person_elastic_search_job_cron-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_CRON_PERSONRECORDSTATUSJOB",
+          "valueFrom": "person_record_status_job_cron-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_CRON_RECORDRESENDINGJOB",
+          "valueFrom": "person_elastic_search_job_cron-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_JOBS_PERSONELASTICSEARCHJOB_PAGESIZE",
+          "valueFrom": "person_elastic_search_job_page_size-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_JOBS_PERSONRECORDSTATUSJOB_DATEOFCHANGEOVERRIDE",
+          "valueFrom": "person_record_status_job_data_of_change_override-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_JOBS_RUNONSTARTUP_EARLIEST",
+          "valueFrom": "jobs-run-on-startup-earliest-sync-{environment}"
+        },
+        {
+          "name": "APPLICATION_JOBS_RUNONSTARTUP_LATEST",
+          "valueFrom": "jobs-run-on-startup-latest-sync-{environment}"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "awslogs-${environment}-tis-sync",
+          "awslogs-region": "eu-west-2",
+          "awslogs-stream-prefix": "awslogs-tis-sync",
+          "awslogs-create-group": "true"
+        }
+      },
+      "portMappings": [
+        {
+          "containerPort": 8101
+        }
+      ],
+      "environment": [
+        {
+          "name": "TITLE",
+          "value": "tis-sync"
+        },
+        {
+          "name": "AWS_REGION",
+          "value": "eu-west-2"
+        }
+      ]
+    }
+  ],
+  "family": "tis-sync-${environment}",
+  "requiresCompatibilities": [
+    "FARGATE"
+  ],
+  "executionRoleArn": "ecsTaskExecutionRole",
```

Comment:
You will need to set the `taskRoleArn` here to provide SQS access for the request queues